### PR TITLE
CFD-357 : Group Categories not transferred as expected

### PR DESCRIPTION
**Pull request by nader77**

### CODE REVIEW

**Comment by nader77** [comment]
The right display of categories in the Quick Post:
category-list-2015-04-27_15

**Comment by nader77** [comment]
Changed the way the categories are sent to AngularJs, Deleted duplicated data sent for each content-type.

**Comment by nader77** [comment]
@HelenaEksler Please review this.

File: capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module [hunk 1]
```diff
@@ -97,13 +97,36 @@ function c4m_restful_quick_post_form($group_id) {
...
   $public_fields = array();
+  $categories = array();
   foreach ($resource_names as $resource_name) {
     $handler = restful_get_restful_handler($resource_name);
     // Send the fields to the app as separate objects by resource
     // (for validation on submit.)
     $field_schema['resources'][$resource_name] = $handler->options('', array('group' => $group_id));
+    // Clean the categories values from the field Schema (Duplicated values).
+    // Order them and assign them to a separate variable.
+    foreach($field_schema['resources'][$resource_name] as $field_name => $values) {
```
Comment (HelenaEksler) [inline capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module]: ``foreach($field_schema...`` => ``foreach ($field_schema...``

**Comment by HelenaEksler** [comment]
@nader77 Looks ok. Just fix this minor thing and assign to Peter.

**Comment by nader77** [comment]
@MPParsley @zero2one Please review this as soon as you can. Thanks.

**Pull request merged**